tests/unit/VCRStorageBucketTest.php

```diff
@@ -11,8 +11,11 @@ class VCRStorageBucketTest extends TestCase
 	public function setup(): void
 	{
 		parent::setUp();
-		$authHeader = ['Authorization' => "Bearer {$_ENV['SERVICE_ROLE']}"];
-		$this->client = new  \Supabase\Storage\StorageClient("https://{$_ENV['PROJECT_REF']}.supabase.co/storage/v1", $authHeader);
+		$dotenv = \Dotenv\Dotenv::createUnsafeImmutable(dirname(__DIR__, 2), '.env.test');
```

Comment:
We should use a path here e.g. __DIR__ . '/../../env.test'; vs dirname()

Reply:
Done